python/rmm/rmm/_lib/memory_resource.pyx

```diff
@@ -1026,32 +1089,27 @@ cdef _per_device_mrs = defaultdict(CudaMemoryResource)
 cpdef void _initialize(
     bool pool_allocator=False,
     bool managed_memory=False,
+    bool system_memory=False,
```

Comment:
I'd like opinion of Python experts (@wence- @vyasr) about whether we want to add another initialization option like this. it's possible to change the default resource to a SAM MR without going through this path, and we don't want to add options for every possible MR here.

Reply:
Yeah not pretty, would love to hear better options.

Reply:
Reverted.